documentation/docs/04-hooks.md

```diff
@@ -85,3 +85,19 @@ export function getSession(request) {
 ```
 
 > `session` must be serializable, which means it must not contain things like functions or custom classes, just built-in JavaScript data types
+
+### serverFetch
+
+This function provides possibilities to replace `fetch` on the server-side for the external API requests. It's beneficial if you want to retrieve resources differently from what it is doing on the browser. For example, if you want to change URL or headers for server-side only.
+
+```ts
+type ServerFetch = (req: Request) => Promise<Response>;
+```
+
+```js
+/** @type {import('@sveltejs/kit').ServerFetch} */
+export async function serverFetch(request) {
+	const data = await getDataMethod(request);
```

Comment:
you could do this in one line:
```
return await getDataMethod(request);
```

or did you mean for the next line to be `return new Response(data)` as in Rich's example? https://github.com/sveltejs/kit/issues/1417#issuecomment-840037007

it's unclear what `getDataMethod` is here though. maybe we should show a more realistic example such as changing the URL

Reply:
> maybe we should show a more realistic example such as changing the URL

ok, I also thought about it.